### ISSUE TITLE
Fresh install panic reboot after going trough the first setup

### ISSUE THREAD

**Issue by jeroenvdd** [jomjol/AI-on-the-edge-device#2333]
### The Problem

every 5 mins this one is dropping. I haven't changed any config / anything special i just did the first setup and after the reboot this keeps happening every 5 min

### Version

Release: v15.1.1 (Commit: 03c84a1+)

### Logfile

```shell
[0d00h00m05s] 2023-04-18T13:19:05 <WRN> [MAIN] Reset reason: Exception/panic
[0d00h00m05s] 2023-04-18T13:19:05 <WRN> [MAIN] Device was rebooted due to a software exception! Log level is set to DEBUG until the next reboot. Flow init is delayed by 5 minutes to check the logs or do an OTA update
[0d00h00m05s] 2023-04-18T13:19:05 <WRN> [MAIN] Keep device running until crash occurs again and check logs after device is up again
[0d00h00m05s] 2023-04-18T13:19:05 <INF> [LOGFILE] Set log level to DEBUG
[0d00h00m05s] 2023-04-18T13:19:05 <INF> [WLANINI] SSID: Home_WIFI
[0d00h00m05s] 2023-04-18T13:19:05 <INF> [WLANINI] Password: XXXXXXXX
[0d00h00m05s] 2023-04-18T13:19:05 <INF> [WLANINI] RSSIThreshold: 0
[0d00h00m05s] 2023-04-18T13:19:05 <INF> [MAIN] WLAN config loaded, init WIFI...
[0d00h00m05s] 2023-04-18T13:19:05 <INF> [WIFI] Automatic interface config --> Use DHCP service
[0d00h00m05s] 2023-04-18T13:19:06 <INF> [WIFI] Set hostname to: watermeter
[0d00h00m05s] 2023-04-18T13:19:06 <INF> [WIFI] Init successful
[0d00h00m07s] 2023-04-18T13:19:08 <INF> [MAIN] PSRAM size: 8388608 byte (8MB / 64MBit)
[0d00h00m07s] 2023-04-18T13:19:08 <INF> [MAIN] Total heap: 4175458 byte
[0d00h00m07s] 2023-04-18T13:19:08 <INF> [MAIN] Camera info: PID: 0x26, VER: 0x42, MIDL: 0x7f, MIDH: 0xa2
[0d00h00m07s] 2023-04-18T13:19:08 <INF> [MAIN] Device info: CPU cores: 2, Chip revision: 3
[0d00h00m07s] 2023-04-18T13:19:08 <INF> [MAIN] SD card info: Name: SS08G, Capacity: 7580MB, Free: 7556MB
[0d00h00m07s] 2023-04-18T13:19:08 <INF> [WIFI] Connected to: Home_WIFI, RSSI: -85
[0d00h00m09s] 2023-04-18T13:19:10 <INF> [MAIN] Initialization completed successfully! Starting flow task ...
[0d00h00m11s] 2023-04-18T13:19:12 <INF> [WIFI] Assigned IP: 192.168.0.72
[0d00h00m35s] 2023-04-18T13:19:39 <INF> [SNTP] Time is synced with NTP Server pool.ntp.org: 2023-04-18 13:19:39
[0d00h05m09s] 2023-04-18T13:24:12 <DBG> [PSRAM] Allocated 921600 bytes in PSRAM for 'C IMG BASIS->CImageBasis (rawImage)'
[0d00h05m11s] 2023-04-18T13:24:14 <DBG> [PSRAM] Allocated 128004 bytes in PSRAM for 'ALIGN->AlgROI'
[0d00h05m11s] 2023-04-18T13:24:14 <DBG> [PSRAM] Allocated 819200 bytes in PSRAM for 'TFLITE->tensor_arena'
[0d00h05m11s] 2023-04-18T13:24:14 <DBG> [TFLITE] CTfLiteClass::LoadModel
[0d00h05m11s] 2023-04-18T13:24:14 <DBG> [TFLITE] CTfLiteClass::ReadFileToModel: /sdcard
[0d00h05m11s] 2023-04-18T13:24:14 <ERR> [PSRAM] Failed to allocate 0 bytes in PSRAM for 'TFLITE->modelfile'!
[0d00h05m11s] 2023-04-18T13:24:14 <ERR> [TFLITE] CTfLiteClass::ReadFileToModel: Can't allocate enough memory: 0
[0d00h05m11s] 2023-04-18T13:24:14 <DBG> [HEAP] CTfLiteClass::ReadFileToModel Heap Total: 2268574 | SPI Free: 2205939 | SPI Large Block: 2162688 | SPI Min Free: 2205423 | Int Free: 62635 | Int Large Block: 59392 | Int Min Free: 58735
[0d00h05m11s] 2023-04-18T13:24:14 <ERR> [CNN] Can't load tflite model -> Init aborted!
[0d00h05m11s] 2023-04-18T13:24:14 <DBG> [HEAP] getNetworkParameter-LoadModel Heap Total: 2268574 | SPI Free: 2205939 | SPI Large Block: 2162688 | SPI Min Free: 2205423 | Int Free: 62635 | Int Large Block: 59392 | Int Min Free: 58735
[0d00h05m11s] 2023-04-18T13:24:14 <DBG> [PSRAM] Freeing memory in PSRAM used for 'TFLITE->modelfile'...
[0d00h05m11s] 2023-04-18T13:24:14 <DBG> [PSRAM] Freeing memory in PSRAM used for 'TFLITE->tensor_arena'...
[0d00h05m11s] 2023-04-18T13:24:14 <DBG> [PSRAM] Allocated 819200 bytes in PSRAM for 'TFLITE->tensor_arena'
[0d00h05m11s] 2023-04-18T13:24:14 <DBG> [TFLITE] CTfLiteClass::LoadModel
[0d00h05m11s] 2023-04-18T13:24:14 <DBG> [TFLITE] CTfLiteClass::ReadFileToModel: /sdcard/config/ana-cont_1105_s2_q.tflite
[0d00h05m11s] 2023-04-18T13:24:14 <DBG> [PSRAM] Allocated 53328 bytes in PSRAM for 'TFLITE->modelfile'
[0d00h05m11s] 2023-04-18T13:24:14 <DBG> [TFLITE] CTfLiteClass::MakeAllocate
[0d00h05m11s] 2023-04-18T13:24:14 <DBG> [PSRAM] Freeing memory in PSRAM used for 'TFLITE->modelfile'...
[0d00h05m11s] 2023-04-18T13:24:14 <DBG> [PSRAM] Freeing memory in PSRAM used for 'TFLITE->tensor_arena'...
[0d00h05m11s] 2023-04-18T13:24:14 <DBG> [PSRAM] Allocated 3072 bytes in PSRAM for 'C IMG BASIS->CImageBasis (ROI ana1)'
[0d00h05m11s] 2023-04-18T13:24:14 <DBG> [PSRAM] Allocated 22707 bytes in PSRAM for 'C IMG BASIS->CImageBasis (ROI ana1 original)'
[0d00h05m11s] 2023-04-18T13:24:14 <INF> [LOGFILE] Set log level to ERROR
[0d00h05m11s] 2023-04-18T13:24:14 <DBG> [TFLITE SERVER] ----------------------------------------------------------------
[0d00h05m11s] 2023-04-18T13:24:14 <INF> [TFLITE SERVER] Round #1 started
[0d00h05m20s] 2023-04-18T13:24:24 <DBG> [PSRAM] Freeing memory in PSRAM used for 'C IMG BASIS->CImageBasis (zwImage, 0)'...
[0d00h05m21s] 2023-04-18T13:24:24 <DBG> [PSRAM] Allocated 921600 bytes in PSRAM for 'C IMG BASIS->CImageBasis (ImageTMP)'
[0d00h05m43s] 2023-04-18T13:24:47 <DBG> [PSRAM] Freeing memory in PSRAM used for 'C IMG BASIS->CImageBasis (ImageTMP, 921600)'...
[0d00h00m04s] 2023-04-18T11:24:54 <INF> [MAIN] =================================================
[0d00h00m04s] 2023-04-18T11:24:54 <INF> [MAIN] ==================== Start ======================
[0d00h00m04s] 2023-04-18T11:24:54 <INF> [MAIN] =================================================
[0d00h00m04s] 2023-04-18T11:24:54 <INF> [SDCARD] Basic R/W check started...
[0d00h00m04s] 2023-04-18T11:24:54 <INF> [SDCARD] Basic R/W check successful
[0d00h00m04s] 2023-04-18T11:24:54 <INF> [SNTP] TimeServer not defined, using default: pool.ntp.org
[0d00h00m04s] 2023-04-18T11:24:54 <INF> [SNTP] Configuring NTP Client...
[0d00h00m04s] 2023-04-18T13:24:54 <INF> [SNTP] Time zone set to CET-1CEST,M3.5.0,M10.5.0/3
[0d00h00m04s] 2023-04-18T13:24:54 <INF> [SNTP] Time is already set: 2023-04-18 13:24:54
[0d00h00m04s] 2023-04-18T13:24:54 <INF> [MAIN] CPU frequency: 160 MHz
[0d00h00m05s] 2023-04-18T13:24:54 <INF> [SDCARD] Folder/file presence check started...
[0d00h00m05s] 2023-04-18T13:24:54 <INF> [SDCARD] Folder/file presence check successful
[0d00h00m05s] 2023-04-18T13:24:54 <INF> [MAIN] Tag: 'v15.1.1', Release: v15.1.1 (Commit: 03c84a1+), Date/Time: 2023-03-23 20:40, Web UI: Release: v15.1.1 (Commit: 03c84a1+)
[0d00h00m05s] 2023-04-18T13:24:54 <WRN> [MAIN] Reset reason: Exception/panic
[0d00h00m05s] 2023-04-18T13:24:54 <WRN> [MAIN] Device was rebooted due to a software exception! Log level is set to DEBUG until the next reboot. Flow init is delayed by 5 minutes to check the logs or do an OTA update
[0d00h00m05s] 2023-04-18T13:24:54 <WRN> [MAIN] Keep device running until crash occurs again and check logs after device is up again
[0d00h00m05s] 2023-04-18T13:24:54 <INF> [LOGFILE] Set log level to DEBUG
[0d00h00m05s] 2023-04-18T13:24:54 <INF> [WLANINI] SSID: Home_WIFI
[0d00h00m05s] 2023-04-18T13:24:54 <INF> [WLANINI] Password: XXXXXXXX
[0d00h00m05s] 2023-04-18T13:24:54 <INF> [WLANINI] RSSIThreshold: 0
[0d00h00m05s] 2023-04-18T13:24:55 <INF> [MAIN] WLAN config loaded, init WIFI...
[0d00h00m05s] 2023-04-18T13:24:55 <INF> [WIFI] Automatic interface config --> Use DHCP service
[0d00h00m05s] 2023-04-18T13:24:55 <INF> [WIFI] Set hostname to: watermeter
[0d00h00m05s] 2023-04-18T13:24:55 <INF> [WIFI] Init successful
[0d00h00m07s] 2023-04-18T13:24:57 <INF> [MAIN] PSRAM size: 8388608 byte (8MB / 64MBit)
[0d00h00m07s] 2023-04-18T13:24:57 <INF> [MAIN] Total heap: 4175458 byte
[0d00h00m07s] 2023-04-18T13:24:57 <INF> [MAIN] Camera info: PID: 0x26, VER: 0x42, MIDL: 0x7f, MIDH: 0xa2
[0d00h00m07s] 2023-04-18T13:24:57 <INF> [MAIN] Device info: CPU cores: 2, Chip revision: 3
[0d00h00m07s] 2023-04-18T13:24:57 <INF> [MAIN] SD card info: Name: SS08G, Capacity: 7580MB, Free: 7556MB
[0d00h00m07s] 2023-04-18T13:24:57 <INF> [WIFI] Connected to: Home_WIFI, RSSI: -83
[0d00h00m09s] 2023-04-18T13:24:59 <INF> [MAIN] Initialization completed successfully! Starting flow task ...
[0d00h00m09s] 2023-04-18T13:24:59 <INF> [WIFI] Assigned IP: 192.168.0.72
[0d00h00m28s] 2023-04-18T13:25:18 <DBG> [OTA FILE] log_get_last_part_handler
```

### Expected Behavior

it remaining up :)

**Comment by jeroenvdd** [comment]
it was the blank model bug apparently, i tought this was fixed but when i looked at the config it seemed to be still blank

**Comment by hilsonp** [comment]
For the record, I got into the exact same situation on the first device I tried to setup.
After finding this page, I finally found why the esp32-cam was crashing every 5 minutes.

It may be good to add a protection if the model is not set. An indication in the UI would be even better or a note in the "Graphical Configuration" that this setting MUST be set. I completely overlooked that page as I thought I would first try with default values ;-)

For other that are finding this page by searching on the crash message, the model is set here:
Settings > Configuration > Digits > Model
